src/Servers/IIS/IIS/src/Core/IISHttpServer.cs

```diff
@@ -163,7 +166,8 @@ private static NativeMethods.REQUEST_NOTIFICATION_STATUS HandleRequest(IntPtr pI
             }
         }
 
-        private static bool HandleShutdown(IntPtr pvRequestContext)
+        [UnmanagedCallersOnly(CallConvs = new[] { typeof(CallConvCdecl) })]
+        private static bool HandleShutdown(nint pvRequestContext)
```

Comment:
I can't recall off the top of my head, but I believe `bool` is still non-blittable in this case so I would instead use `byte` for a C++ bool or `int` for a Win32 bool scenario.

Reply:
Good catch. You should see tests failing because of this. This should use `int`.

Reply:
Done!

Reply:
This should be an analyzer error or something.